source/Playnite/HdrUtilities.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Playnite.Native;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Playnite
+{
+    public class HdrUtilities
+    {
+        /// <summary>
+        /// Determines if HDR is supported on the primary display
+        /// </summary>
+        /// <returns>True if HDR is supported on the primary display, false if unknown or not supported</returns>
+        public static bool IsHdrSupported()
+        {
+            DISPLAYCONFIG_PATH_TARGET_INFO? targetInfo = GetPrimaryDisplayTargetInfo();
+            if (!targetInfo.HasValue)
+            {
+                return false;
+            }
+
+            (bool? isHdrSupported, _) = GetHdrStatus(targetInfo.Value);
+            return isHdrSupported.GetValueOrDefault(false);
+        }
+
+        /// <summary>
+        /// Determines if HDR is enabled on the primary display
+        /// </summary>
+        /// <returns>True if HDR is enabled on the primary display, false if unknown or not supported</returns>
+        public static bool? IsHdrEnabled()
+        {
+            DISPLAYCONFIG_PATH_TARGET_INFO? targetInfo = GetPrimaryDisplayTargetInfo();
+            if (!targetInfo.HasValue)
+            {
+                return false;
+            }
+
+            (_, bool? isHdrEnabled) = GetHdrStatus(targetInfo.Value);
+            return isHdrEnabled;
+        }
+
+        /// <summary>
+        /// Enables/disables HDR on the primary display
+        /// </summary>
+        /// <param name="enable">True if enabling HDR, false if disabling HDR</param>
+        public static void EnableHdr(bool enable)
```

Comment:
Should be called `SetHdr`. Calling enable something with false option looks weird.